### ISSUE TITLE
Add Convenience Method `TestChromosome::toTestSuite`

### ISSUE THREAD

**Issue by schweikl** [VoglSebastian/evosuite#10]
### Context
Currently, whenever a `TestChromosome` is to be used in contexts where `TestSuiteChromosomes` are required, one has to resort to using wrapper classes or similar tricks to make it work. This is kind of painful due to the need of wrapping and unwrapping all the time. Instead, it might be a better idea to provide a convenience method `TestSuiteChromosome toTestSuite()` in class `TestChromosome` that converts the current `TestChromosome` instance into a corresponding `TestSuiteChromosome`. For example:
```
public final class TestChromosome extends AbstractTestChromosome<TestChromosome>  {
	public TestSuiteChromosome toTestSuite() {
		final var suite = new TestSuiteChromosome();
		suite.addTest(this);
		return suite;
	}
        // ...
}
```
However, at this point it is unclear whether this would be the proper way to do the conversion. Related Issue: #9 

Alternatively/additionally, it might also be conceivable to add `TestChromosome` convenience methods to classes such as `TestSuiteFitnessFunction`. For example, to compute the fitness of a `TestChromosome` using a `TestSuiteFitnessFunction`:
```
public abstract class TestSuiteFitnessFunction extends FitnessFunction<TestSuiteChromosome> {
	public double getFitness(final TestChromosome individual) {
		return this.getFitness(individual.toTestSuite());
	}
        // ...
}
```
Is this a good idea or not?

**Comment by VoglSebastian** [comment]
I have added a function TestChromosome::toSuite in commit b903ba9d98a5e8607b37d0f88c5d1213136daa86 .

This issue can be closed?

**Comment by schweikl** [comment]
Closed because issue was resolved by b903ba9
